### ISSUE TITLE
Error in importing time does not make zenroom fails

### ISSUE THREAD

**Issue by matteo-cristino** [dyne/Zenroom#781]
Using the contract
```gherkin
Given I have a 'time dictionary' named 'num'

then print the data
```
with input
```json
{
	"num": {
		"key_1": "1.2",
		"key_2": "100"
	}
}
```
returns
```json
{
   "num": {
      "key_2": 100
   }
}
```
as output, but also
```
[
"ZENROOM JSON LOG START",
"Release version: v4.6.0",
" Build commit hash: 69645e9f",
" Memory manager: mimalloc",
" ECDH curve is SECP256K1",
" ECP curve is BLS381",
"[W] Zencode is missing version check, please add: rule check version N.N.N",
"[!] Could not parse unix timestamp 1.2",
" Zenroom execution completed.",
"[*] Zenroom teardown.",
" Memory used: 410 KB",
"ZENROOM JSON LOG END" 
]
```
in the logs.
(time can not be floating numbers)

**Comment by jaromil** [comment]
better use another term for the error message:

Could not read unix timestamp

since parse is confusing as many reference the parser

**Comment by matteo-cristino** [comment]
The main I was referencing to here is not the error message, but the fact that zenroom ended without throwing an error even if a error occoured.

**Comment by matteo-cristino** [comment]
fix in #780. Both the error not thrown and the error message
